.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 4
```

Comment:
Do we need this file as part of this PR?

Reply:
I mean, editorconfig helps with consistency on different editors (I personally like tab-indentation more, but I set it up to whatever was used in the project). I can remove it if you want.

Reply:
No it should be fine.